wf/ja-jp/Content/Topics/Workflows/use-case-identity-creation-conflicts.htm

```diff
@@ -414,4 +414,4 @@ <h2>ガイドラインと制限事項</h2>
             </div>
         </div>
```

Comment:
This is odd. I don't see the feedback widget change here... need to look into this.

Reply:
I found about 30 such files. I think these are stale/orphans, investigating.

Reply:
good catch! file in a long-lived branch does not have a feedback update https://github.com/okta/okta-help/blob/docs_translations_gh_WF/wf/en-us/Content/Topics/Workflows/workflow-connect-your-applications.htm
seems like stale files.
I think the next step for improving build quality can be removing the output directory before we copy a new build. I remember it was that way in flare, but it also removed some files which we didn't build